buildSrc/subprojects/build-update-utils/src/main/kotlin/gradlebuild/buildutils/tasks/UpdateReleasedVersions.kt

```diff
@@ -44,7 +44,7 @@ abstract class UpdateReleasedVersions : DefaultTask() {
         val releasedVersions = Gson().fromJson(releasedVersionsFileObject.reader(), ReleasedVersions::class.java)
         val newReleasedVersions = updateReleasedVersions(currentReleasedVersionValue, releasedVersions)
 
-        releasedVersionsFileObject.appendText(JsonBuilder(newReleasedVersions).toPrettyString())
+        releasedVersionsFileObject.writeText(GsonBuilder().setPrettyPrinting().create().toJson(newReleasedVersions))
```

Comment:
Can you add a test as well, so this does not break again?

Reply:
Good call @wolfs. I added an integration test.